### DISCUSSION TITLE
Feature Request: Floating labels

### DISCUSSION THREAD

**Discussion by LiveNathan** [saadeghi/daisyui#2726] [Ideas / feature requests]
Like https://preline.co/docs/input.html#floating-label but I'd prefer to use DaisyUI.

![2024-01-03 at 17 28 31](https://github.com/saadeghi/daisyui/assets/60303904/362af6d1-dfbf-425a-b36e-3add2f879a17)

**Comment by saadeghi** [comment]
duplicate of #298
